colortv/docs/androidTVSDK.md

```diff
@@ -792,19 +801,196 @@ Check our example [item layouts](https://github.com/color-tv/android-SampleApp/t
+* **Device:** ALL
+<br /><br />
+
+#### **ID:** ctv_upNextContainer
+* **View type:** Any view extending the ViewGroup class (eg. `LinearLayout`, `FrameLayout`)
+* **Description:** Used to automatically locate UpNext fragment at a bottom of a screen when it's parent is `RelativeLayout` or `FrameLayout`. If the view is an instance of `LinearLayout` and contains a views with ids: `ctv_space` and `ctv_upNext`, then it sets weight of that elements to take half of a screen on TVs and tablets, and full screen on mobiles. 
+* **Device:** ALL
+<br /><br />
+
+#### **ID:** ctv_space
+* **View type:** Any view extending the View class, however recommended `Space` class.
+* **Description:** If is a child of `ctv_upNextContainer` `LinearLayout` and sibling of `ctv_upNext` then used to change width of up next to take just half of a screen on TVs and tablets.
+* **Device:** ALL
+<br /><br />
+
+</details>
+
+#### Programmable customization 
+
+In order to change look and feel of default UpNext layout, you can use `ColorTvUpNextUiCustomizer` class which can be taken from `ColorTvUpNextFragment`:
+
+```java
+ColorTvUpNextUiCustomizer uiCustomizer = upNextFragment.getUiCustomizer();
+```
+
+`ColorTvUpNextUiCustomized` provides a following bunch of methods:
+
+```java
+uiCustomizer.setSidebarColorRes(@ColorRes int resId);
+uiCustomizer.setSidebarColor(@ColorInt int sideBarColor);
+uiCustomizer.setBackgroundColorRes(@ColorRes int resId);
+uiCustomizer.setBackgroundColor(@ColorInt int backgroundColor);
+uiCustomizer.setTitleColorRes(@ColorRes int resId);
+uiCustomizer.setTitleColor(@ColorInt int textColor);
+uiCustomizer.setDescriptionColorRes(@ColorRes int resId);
+uiCustomizer.setDescriptionColor(@ColorInt int descriptionColor);
+```
+
+Visualization of all changable elements is outlined on the following image:
+
+![UpNext default layout](images/up_next_showcase_2.png)
+
+### Both Recommendation Center and UpNext related customization
+
+Following config methods has influance on both Recommendation Center and UpNext customization:
+
+```java
+recommendationConfig.setFont(Device device, Typeface typeface);
+recommendationConfig.resetToDefault();
+```
+
+#### setFont(Device device, Typeface typeface)
+
+This method is used to set a custom font for a specified device type. It only works if you don't use a custom item layout.
+
 #### resetToDefault()
 
 This method is used to reset all settings to defaults.
 
 For an example of the usage of all of the above methods, check our [custom recommendation center sample](https://github.com/color-tv/android-SampleApp/blob/master/SampleApp/app/src/main/java/com/colortv/sample/CustomRecommendationCenterActivity.java).
+
+#### Custom Timer Layout
+
+You can include into your custom layouts a view which displays progress until recommendation will be chosen automatically. You can do it in 3 different ways: 
```

Comment:
`You can include into your custom layouts a view which` -> `You can include  a view in your custom layouts, which`